### ISSUE TITLE
Create automated sitemap.xml generation for policy/guidance pdfs

### ISSUE THREAD

**Issue by jason-upchurch** [fecgov/fec-cms#3598]
@jason-upchurch commented on [Fri Feb 28 2020](https://github.com/fecgov/fec-cms/issues/3581)

### Summary
From https://github.com/fecgov/fec-cms/issues/3554, we need to build automated sitemap generation based on existing/added/deleted content and create update policy/schedule, e.g., once per day to be consistent with search.gov

**No longer blocked:** This ticket is presently blocked by fecgov/openFEC#3580, which creates the strategy for dealing with policy/guidance documents in an unambiguous way to enable automatic sitemap generation (which will enable automatic index updating).

When working on this ticket, we need to consider: https://github.com/fecgov/fec-cms/issues/3280

### Completion criteria
- [x] sitemap.xml is generated correctly based on established convention/strategy
  - _current strategy is to provide a hard-coded list of resources that need to be searchable_
- [ ] search.gov uses domains we specify (e.g., change `proxy` url to "correct one")
- [ ] provide search.gov url from github so that they update sitemap from repo
- [ ] research `robots.txt` and indexing multiple sitemaps
- [ ] research domain partitioning such that global search does not pollute guidance search
- [ ] research options for a "highly automated process" for generating sitemaps such that people will not need to maintain them by hand

### Technical considerations
- ~ATO~
- ~AWS/cloud.gov v server~
- We appear to house all of our scheduled tasks in openFEC repo, thus transferred this issue to openFEC and will use existing framework

---

@jason-upchurch commented on [Tue Mar 03 2020](https://github.com/fecgov/fec-cms/issues/3581#issuecomment-593956558)

Related issue opened with cloud.gov to consider AWS Lambda or Scheduled Task: https://github.com/cloudfoundry-community/s3-broker/issues/38

---

@jason-upchurch commented on [Tue Mar 03 2020](https://github.com/fecgov/fec-cms/issues/3581#issuecomment-594037247)

@lbeaufort advised in sprint planning to use caution with respect to this work as it pertains to ATO. 

cc @wjiangFEC @PaulClark2 

---

@jason-upchurch commented on [Tue Mar 03 2020](https://github.com/fecgov/fec-cms/issues/3581#issuecomment-594044531)

requesting ATO for review. Issue open with cloud.gov to check feasibility of deploying cloud.gov only solution. Investigating contingency plans, i.e., server based, in the event cloud.gov is unable to support application.

**Comment by JonellaCulmer** [comment]
Duplicate ticket, so closing in favor of https://github.com/fecgov/fec-cms/issues/3804
